book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rent_book(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] In the `rent_book` method, you don't have to manually push `rental` to the `@rentals` array. This is done automatically by the `Rental` class when an instance of the `Rental` class is created.
Instead, what you should do is pass in the `person` and the `date`, then the book itself should be sent as a parameter to create the Rental. See below 👇 
``` ruby
def rent_book(person, date)
  Rental.new(date, person, self)
end
```

Reply:
Okay, thank you for your insight. I will work on it accordingnly